Proposal-EDGI_ML_project.md

```diff
@@ -0,0 +1,165 @@
+
+**Resources** :
+
+I may need some help from a Machine Learning expert when working on ensemble models(if required). Depending on the size of the data and the complexity of the algorithm, I may require cloud compute engine instances to train and test models.
+
+**Setup** :
+
+I use the [Anaconda](https://www.continuum.io/) distribution for Python on my machine. I have both Python 2.7 and 3.5 on my machine in different work environments and can work with either version depending on the requirement.
+
+Tensorflow for Windows is only available for Python versions 3.5x. I may need to set up a virtual machine if a task requires the use of Tensorflow with Python 2.7 or I can run it on a cloud engine instance.
+
+**Ongoing involvement** :
+
+I support the clean energy movement and believe that we need to tackle environmental problems before they turn into major catastrophes. I would love to continue working with EDGI and other initiatives which use Data Science, Machine Learning etc. to solve problems and protect the environment from ignorant administration.
+
+### **Student Experience**
+
+**Previous Experience** :
+
+Undergraduate Researcher, Visvesvaraya National Institute of Technology
+
+- Relation Extraction from Scientific Text using Recurrent Neural Networks :
+  - Currently working with a team to make a relation extractor for scientific text using deep recurrent neural networks.
+  - Working with Tensorflow and exploring its features.
+- Transient Instability Prediction using Decision Trees :
+  - Currently working on a decision tree classifier which will determine the stability of a system based on its electrical and mechanical properties.
+  - Plan to use this model on the data from a certain region of the Indian National Grid.
+
+Data Science Intern, [Numerify Software India Ltd.](http://www.numerify.com/) (May - July 2016)
+
+- Worked on linear regression and classifier models for IT Service Management Data problems.
+- Added mechanism for calculation of model performance metrics.
+- Also, worked on classifiers for sentiment analysis.
+
+Chapter Head, [Uddeshya India](http://www.uddeshyaindia.org/) (July 2015 - July 2016)
+
+- Appointed as the head for Uddeshya (NGO), Nagpur and led the chapter team for a period of one year.
+
+**Open Source Project(s) you are working on or would like to**:
+
+I have recently started working on scikit-learn issues and will come up with solutions that can be merged with the main repository.
+
+After I have gained sufficient expertise in Machine and Deep Learning, I would like to contribute to Tensorflow.
+
+**Teamwork** : I have some experience working with teams in an industrial as well as an academic environment. I was a part of the data science team during my internship last summer. I have worked with some of my batchmates(under the guidance of our professors) as a team for projects and competitions.
```

Comment:
@dcwalk Thanks for answering my question about the teamwork field. I've added my experience of working on teams.